framework/src/materials/GenericConstantMaterial.C

```diff
@@ -46,6 +46,12 @@ GenericConstantMaterial::GenericConstantMaterial(const InputParameters & paramet
 }
 
 void
+GenericConstantMaterial::initQpStatefulProperties()
```

Comment:
What would you suggest as a better solution?  This material declares properties that are used as stateful in various tests - so as per original MOOSE behavior (which is not changed in this PR) - the material class needs to implement/override stateful prop initialization function.